plover/gui_qt/main.py

```diff
@@ -13,7 +14,8 @@
 
 from plover import _, __name__ as __software_name__, __version__, log
 from plover.oslayer.keyboardcontrol import KeyboardEmulation
-
+from plover.oslayer.config import CONFIG_DIR
+import os
```

Comment:
Keep the blank line between standard and GUI specific imports (and keep them sorted). Move the `os` import where it belongs (together with the other Python standard modules at the top).